docs/dev/build_webassembly.md

```diff
@@ -24,7 +24,7 @@ $ docker run -it --rm -v `pwd`:/openvino emscripten/emsdk bash
 3. (CMake configure) Run cmake configure step using helper emscripten command:
 ```sh
 $ mkdir build && cd build
-$ emcmake cmake -DCMAKE_BUILD_TYPE=Release /openvino
+$ emcmake cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_MLAS_FOR_CPU=OFF /openvino
```

Comment:
it's automatically turned off according to the changes https://github.com/openvinotoolkit/openvino/pull/19005/files, is not it?